### DISCUSSION TITLE
Issue Accessing GitHub Education Teacher Benefits

### DISCUSSION THREAD

**Discussion by opitzl** [community/community#147400] [GitHub Education]
### Hi everyone,

Dear GitHub Support Team,
I was accepted as a teacher for the GitHub Education program on December 5th, but I have yet to receive the associated benefits even though it's written 'your benefits are now available. ' Specifically, I am unable to access GitHub Copilot, which I understand should be included as part of the program.
Many of my colleagues who were accepted to the same program automatically received these benefits, including access to GitHub Copilot, without any issues. I believe there may have been an oversight or delay in my case.
Could you kindly assist in resolving this matter? If any additional steps or information is required from my end, please let me know, and I’ll be happy to provide it.
Thank you for your time and support. I look forward to your response.

Best regards,
Lennart Opitz

**Comment by davevad93** [comment]
Hi @opitzl ,

If you can't access your free `Copilot` subscription by activating it in your [copilot settings](https://github.com/settings/copilot), you should try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue (I know you are a teacher, but it's the only option available).

**Reply by opitzl** [reply]
Many thanks for this suggestion. I will do that.

**Reply by davevad93** [reply]
You're welcome @opitzl 👍
